internal/hud/renderer.go

```diff
@@ -202,6 +201,12 @@ func isCrashing(res view.Resource) bool {
 }
 
 func bestLogs(res view.Resource) string {
+	if res.IsTiltfile && res.CrashLog != "" {
```

Comment:
Hmm, I'd be surprised if it makes sense to have the crash log preempt the build log for tiltfiles.

I'd think we'd always want to show all the tiltfile output and any error that occurred, rather than only showing the error when there's an error, and then just not bother touching the CrashLog field.

The segregation between build and crash logs for k8s services is due to wanting to smooth things out across pod restarts, which isn't really relevant to tiltfile execution.

Or maybe I've misunderstood this behavior?

Reply:
Yeah this is a good point, let me experiment with this some more.